picard/ui/options/cover_processing.py

```diff
@@ -80,6 +113,9 @@ def __init__(self, parent=None):
             spinbox.setEnabled(checkbox.isChecked())
             checkbox.stateChanged.connect(self._update_resize_spinboxes)
 
+    def _add_rich_text(self, text):
+        return "<FONT COLOR=black>" + text + "</FONT>"
```

Comment:
Fixing color here isn't a good idea, it might cause issues depending on themes.
This shouldn't be needed.